### PR TITLE
SUBMARINE-868. Add save model to pysubmarine ModelsClient

### PR DESCRIPTION
**Pull request by jeff-901**
### What is this PR for?
Add save model function to submarine-sdk model management.

### What type of PR is it?
Feature

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-868

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? Yes

### CODE REVIEW

**Comment by jeff-901** [comment]
@ByronHsu Help me review the code. Thanks.

File: submarine-sdk/pysubmarine/submarine/models/utils.py [hunk 1]
```diff
@@ -76,3 +77,12 @@ def get_worker_index():
         worker_index = "worker-0"
 
     return worker_index
+
+
+def get_ps():
```
Comment (ByronHsu) [inline submarine-sdk/pysubmarine/submarine/models/utils.py:82]: Is it better to be named as `exist_ps`?

Comment (jeff-901) [reply submarine-sdk/pysubmarine/submarine/models/utils.py:82]: Yes, I will change to function name.

**Comment by ByronHsu** [comment]
LGTM

**Pull request closed without merging**